API.yaml

```diff
@@ -930,14 +1057,34 @@ paths:
           type: string
       requestBody:
         description: Query parameters to fetch the tooltips.
-          Usually contains these parameters, but not limited to
-          - entries, array of all the entry (series) IDs
-          - times, array of all the times
         required: true
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/Query'
+              type: object
```

Comment:
Tooltips can be a GET. All the implementations are so far are GET. Let's align it to GET given the current deadline. If needed later on we can change it to POST.